style.css

```diff
@@ -22,11 +30,105 @@ body {
   align-items: center;
   text-align: center;
   color: #28352f;
-  padding-left: 25px;
+  text-decoration: none;
+  padding-left: 10px;
+}
+
+#burger {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 25px;
+  height: 25px;
+  background-color: #f7f8f7;
+  border-radius: 3px;
+}
+
+.hold {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  height: 65%;
+}
+
+.dash {
+  background-color: #422a7c;
+  height: 10%;
+  width: 20px;
+  border-radius: 2px;
+}
+
+#burger:hover {
+  background-color: #494b49;
+}
+
+#burger:active {
+  background-color: #f7f8f7;
+}
+
+#burger:hover > .hold > .dash {
+  background-color: #fff;
 }
 
-.menu {
-  padding-right: 25px;
+#burger:active > .hold > .dash {
+  background-color: #422a7c;
+}
+
+#menu-block {
```

Comment:
- [ ] [OPTIONAL] You could add more margin at the top of the mobile menu, so it can be as close to the Figma design as possible